### ISSUE TITLE
A question about a conclusion on the article

### ISSUE THREAD

**Issue by Sandy-program** [MaxZanella/CLIP-LoRA#11]
Dear Authors, thank you for your interesting work and for releasing the code.
I see your conclusion in the article: "With the exception of EuroSAT, where adapting
solely the vision encoder shows marginally better stability, tuning both encoders concurrently is the most effective
strategy, leading to significant enhancements."
This conclusion is based on the three data sets you listed: (a) ImageNet,(b) Stanford Cars and (c) EuroSAT or based on all the data sets in Figure 2?
In other words, whether some datasets and others, such as the EuroSAT dataset, adjust both the text and visual encoders, did not get the best results.

**Comment by MaxZanella** [comment]
Hello,

Thank you for your interest in our work! This trend was generally consistent across other datasets, and we included EuroSAT for its interesting behavior. However, we did not run this extensive ablation study on all datasets so I could not provide you the same figures for the other datasets. 
You can easily try the different configurations by playing with the --encoder argument ('vision', 'text' or 'both'). Also, keep in mind that the figures provided are for the 4-shot setting. You might want to experiment with varying this as well.

**Comment by Sandy-program** [comment]
Thank you very much for your help and look forward to your better work
